jvector-twenty/src/main/java/io/github/jbellis/jvector/vector/PanamaVectorUtilSupport.java

```diff
@@ -18,122 +18,129 @@
 
 import io.github.jbellis.jvector.vector.types.ByteSequence;
 import io.github.jbellis.jvector.vector.types.VectorFloat;
+import jdk.incubator.vector.FloatVector;
+import jdk.incubator.vector.VectorSpecies;
 
 import java.util.List;
 
-final class PanamaVectorUtilSupport implements VectorUtilSupport {
+class PanamaVectorUtilSupport implements VectorUtilSupport {
+    protected final SimdOps simdOps;
```

Comment:
Is it needed to keep here an instance of `SimdOps`? It looks like previously it was static which is more efficient when no state is required to keep for method calls.

Reply:
We don't need separate SimdOps classes etc, we can move things to VectorSupportUtils implementations?